mypy_json_report.py

```diff
@@ -67,7 +67,7 @@ def _no_command(args: argparse.Namespace) -> None:
 
     This will be hit when the program is called without arguments.
     """
-    print("A subcommand is required. Pass --help for usage info.")
+    print("A subcommand is required. Pass --help for usage info.", sys.stderr)
```

Comment:
**Suggestion:** I think you want to be sure to set that to the `file` argument, otherwise you just print the `sys.stderr` variable:
```
$ mypy-json-report
A subcommand is required. Pass --help for usage info. <_io.TextIOWrapper name='<stderr>' mode='w' encoding='utf-8'>
```

```suggestion
    print("A subcommand is required. Pass --help for usage info.", file=sys.stderr)
```

Reply:
Oops -- good catch, thanks!